src/storage/btree.h

```diff
@@ -192,6 +248,13 @@ struct btree_scan
   DB_VALUE cur_key;		/* current key value */
   bool clear_cur_key;		/* clear flag for current key value */
 
+  //---------------------------------------------------------------------------------------------   
+  /* IMPROVE_RANGE_SCAN_IN_BTREE */
+  bool is_cur_key_decompressed;	/* If true, cur_key is a complete key.
+				 * Otherwise it must be combined with C_page_info.compress_key. */
+  BTREE_PAGE_PREFIX_INFO C_page_info;
```

Comment:
There are variables that were previously used such as C_vpid and C_page.
I think it makes sense to start with a capital "C_" because it is associated with these variables.

I'll keep this name unless there are other important reasons.